ExplorerTests/ExplorerTests.swift

```diff
@@ -21,7 +21,7 @@ class ExplorerTests: XCTestCase {
         super.tearDown()
     }
     
-    func testInitDiveLogViewModel() {
+    func testInitDiveLogViewModel_WithNilImage() {
```

Comment:
Avoid using `_` in names in Swift -> `testInitDiveLogViewModelWithNilImage`

Reply:
Per offline discussion, ok to use _ in tests.